ansible/roles/file-storage/templates/ingress.yml.j2

```diff
@@ -7,15 +7,15 @@ metadata:
     nginx.ingress.kubernetes.io/proxy-send-timeout: "3600"
     nginx.ingress.kubernetes.io/proxy-read-timeout: "3600"
     nginx.ingress.kubernetes.io/proxy-body-size: "{{ INGRESS_MAX_BODY_SIZE|default("2560") }}m"
-    nginx.org/client-max-body-size: "{{ INGRESS_MAX_BODY_SIZE|default("2560") }}m"
+    # necessary to stream requests directly though, otherwise file-service is informed too late and user will not be notified that an upload is in progress
```

Comment:
```suggestion
    # necessary to stream requests directly through, otherwise file-service is informed too late and user will not be notified that an upload is in progress
```